experimental/origin-dapp2/src/queries/Fragments.js

```diff
@@ -141,5 +142,39 @@ export default {
         }
       }
     `
+  },
+  GrowthCampaign: {
+    basic: gql`
+      fragment basicCampaignFields on GrowthCampaign {
+        id
+        name
+        startDate
+        endDate
+        distributionDate
+        status
+        actions {
+          type
+          status
+          rewardEarned {
```

Comment:
nit: Please check with Nick and Stan re standard for currency objects in GraphQL schema. There was a debate for flat vs nested fields. like rewardEarnedAmount and rewardEarnedCurrency vs rewardEnarned { amount, currency } and I don't remember what was decided :)

Reply:
will do 👍

Reply:
I think we decided to flatten it but we haven't gotten around to it yet...

Reply:
Ok leaving it as it is for now then